### PR TITLE
Sorted properties from custom context

### LINKED ISSUE #56: Properties from custom assembly are in random order
When using a context from a custom assembly, the properties are displayed in a random order.  I will prepare a PR to sort them.

### PR DESCRIPTION
**Pull request by RoyChase**
Sorted properties from custom context
Fixed typo in ConnectionDialog label

Fixes #56

### CODE REVIEW

**Review by MaceWindu — APPROVED** [review]
Thanks. Will merge later when we are ready to release new version. Want to check if it is really random or use column ordinal

**Comment by RoyChase** [comment]
It's not column ordinal as it doesn't make a call to the database when generating the tree.  From some test I am doing it looks to be in the order that the properties are in the source (which is logical) but I doesn't always seem to be this way as the context I was originally trying this with has the properties in alphabetic order.  I don't have this to had today but I'll do some more tests to confrm this and that the fix actually works.  Leave it with me.

**Comment by MaceWindu** [comment]
Ah, you are right. Here we load columns using reflection.

Usually it is ordered by declaration order in source code due to implementation, but as it documented [here](https://docs.microsoft.com/en-us/dotnet/api/system.type.getmembers?view=net-5.0) it is not always true :)

File: Source/SchemaGenerator.cs [hunk 1]
```diff
@@ -54,6 +54,7 @@ public IEnumerable<ExplorerItem> GetSchema()
 				.Where(p =>
 					p.GetCustomAttributeLike<ObsoleteAttribute>() == null &&
 					p.PropertyType.MaybeChildOf(typeof(IQueryable<>)))
+				.OrderBy(p => p.Name)
```
Comment (MaceWindu) [inline Source/SchemaGenerator.cs:57]: I suggest to sort by [`ColumnAttribute.Order`](https://github.com/linq2db/linq2db/blob/master/Source/LinqToDB/Mapping/ColumnAttribute.cs#L330-L340) for columns with `HasOrder` and by name for rest

Comment (RoyChase) [reply Source/SchemaGenerator.cs:57]: The original change will sort the tables.  I'm still trying to sort the columns but I will honor the ColumnAttribute if the order is specified.

Comment (MaceWindu) [reply Source/SchemaGenerator.cs:57]: not sure of helpful, but here we apply similar sorting for columns in create table based on Order: https://github.com/linq2db/linq2db/blob/master/Source/LinqToDB/SqlProvider/BasicSqlBuilder.cs#L1111

Comment (RoyChase) [reply Source/SchemaGenerator.cs:57]: Tables and columns shold now sort by name honouring column attribute order if applicable

**Review by MaceWindu — APPROVED** [review]

**Pull request merged**